### ISSUE TITLE
Upgrade Heron to support OpenLayers 2.13.1

### ISSUE THREAD

**Issue by justb4** [heron-mc/heron-mc#430]
OpenLayers 2.13 and beyond has been out for some time. This issue identifies the work to upgrade Heron to support OpenLayers 2.13.1, and if possible also 2.12.

Main work items here are:

- see what has been fixed from 2.12 to 2.13 that was patched in Heron via
https://github.com/heron-mc/heron-mc/blob/master/heron/lib/override-openlayers.js, remove fixed code (potentially make it conditional for 2.12)
- make 2.13.1 available omn CDNJS (if not already there)
- regression testing of all examples and known applications like the KadViewer http://kadviewer.kademo.nl

**Comment by justb4** [comment]
Comment from Davor:

"I use it for more than a year, but:

1. Use latest master form github https://github.com/openlayers/openlayers and build yourself
2. Don't use Heron's override-openlayers.js, as I can remember all issues were resolved (I don't use it)
3. For IE<9 users you have to patch it with this: https://trac.osgeo.org/openlayers/attachment/ticket/2578/07-destroy-controls-check.patch
4. If you want to use smooth transition effects, in Heron.options.map.settings, beside transitionEffect : 'resize' also set tileManager : null, or you will be experiencing some annoying flickering when zooming
5. You will still sometimes experience issue with showing white lines during transitions when zooming out - this is a OL issue: https://github.com/openlayers/openlayers/issues/458, you can see the example at: http://geoportal.dgu.hr/viewer/"

**Comment by dracic** [comment]
One more thing... For Fade-in effects when using single tile layers, I use this:

http://lydonchandra.com/content/openlayers-fade-effect-single-tile-wms-and-no-fade-effect-multi-tile-wms

**Comment by vivkekanwar** [comment]
Thanks Just and Davor and sorry for my late reply.
I tried the ol 2.13.1 version but what problem I have experienced that layers doesn't load  or display at the start up and when I do some activities using mouse, all the layers 
added to the map layout. Any idea how to resolve this.

**Comment by justb4** [comment]
Hi, did you follow the 5 steps that Davor described above?

**Comment by vivkekanwar** [comment]
Yes I have tried these but results are the same, layers doesn't load at start up but after mouse activity or doing layer on off they start displaying.

**Comment by dracic** [comment]
Did you set 'maxExtent', 'center' and 'zoom' property of Heron.options.map.settings? Try to use 'Developer tools' in Google Chrome, to see if anything pops up in 'Console'. Did you use any additional OpenLayers controls - I had some problem ? If you did, try without using them. Without your configuration example we cannot be of much help.

**Comment by vivkekanwar** [comment]
Hiiiii dracis and Just, thanks for the help, problem is solved. I just added tileManager: null in the Heron.options.map.settings and used openlayers 2.13.1 instead of ol 2.12, it worked for me. I am also attaching the image to show the changes made in the js file.
![heron](https://cloud.githubusercontent.com/assets/12725918/8568914/01a9b418-2590-11e5-851b-bbeec32ac4d8.JPG)
